assets/common/environments/python-sdk-v2/context/Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM mcr.microsoft.com/azureml/openmpi4.1.0-ubuntu20.04:{{latest-image-tag}}
 
-RUN apt-get update -y && apt-get install openssh-server openssh-client openssl curl -y
+RUN apt-get update -y && apt-get install libc-bin libc-dev-bin libc6 libc6-dev libcurl3-gnutls tar openssh-server openssh-client openssl curl -y
```

Comment:
why are we updating libc?

Reply:
Should not this be updated and resolved in the base image?

Reply:
If we will not do it here, we will get 10 more vulnerabilites with latest image. Right now both the images are having 0 vuln with these changes in top of base image